plugins/modules/deb_remote.py

```diff
@@ -31,6 +31,22 @@
       - immediate
       - on_demand
       - streamed
+  sync_installer:
+    description:
+      - Sync installer packages
+    type: boolean
```

Comment:
looks like this needs to be `bool`

Reply:
Sorry for overlooking that. I've pushed the change. Let me know if there is anything else.